### DISCUSSION TITLE
Face crop recommendation?

### DISCUSSION THREAD

**Discussion by jysung** [TheLastBen/fast-stable-diffusion#524] [Q&A] [answered]
Has anyone played with how tight to do face crops? Does it affect Dreambooth in any way?

**Comment by TheLastBen** [comment, marked as answer]
you need different perspective, closeups, medium shots and far shots, make the closeups more present

**Reply by jysung** [reply]
If I am training with a resolution of 512, Is there a difference between feeding the model 512 x 512 vs 2048 x 2048?

**Reply by SAC020** [reply]
According to my limited experience, it will generate better quality gens

What I am still not clear about is whether we can mix various resolutions pics, as long as they are square

**Reply by TheLastBen** [reply]
1024px would be way better
